backend/infrahub/support/macro.py

```diff
@@ -1,69 +0,0 @@
-from __future__ import annotations
-
-from typing import Any
-
-from jinja2 import TemplateSyntaxError, meta, nodes
-from jinja2.sandbox import SandboxedEnvironment
-
-ALLOWED_FILTERS = [
-    "abs",
-    "capitalize",
-    "center",
-    "default",
-    "escape",
-    "filesizeformat",
-    "join",
-    "length",
-    "list",
-    "lower",
-    "replace",
-    "reverse",
-    "round",
-    "sort",
-    "sum",
-    "title",
-    "trim",
-    "truncate",
-    "upper",
-    "wordcount",
-]
-
-
-class MacroDefinition:
-    def __init__(self, macro: str) -> None:
-        self.macro = macro
-        self.env = SandboxedEnvironment()
-        self._filters: list[str] = []
-        self._variables: list[str] = []
-        self.template = self._parse_template()
-
-    def _parse_template(self) -> nodes.Template:
-        try:
-            self.env.globals = {}
-            template = self.env.parse(self.macro)
-            if any(node.__class__.__name__ in ["Call", "Import", "Include"] for node in template.body):
```

Comment:
same comment here, are we still checking for that ?

Reply:
This check has been moved to the SDK, I realized that they don't actually work when using string templates regardless. However they will never render anyway, with that in mind I think it's still better that any schema that uses any of these commands would be rejected when loading instead of failing later.